src/lua/debug.lua

```diff
@@ -0,0 +1,28 @@
+local debug = require('debug')
+
+local function sourcepath(lvl)
+    local info = debug.getinfo(lvl or 2, 'S')
+    local source = info and info.source
+    return source and source:sub(1, 1) == '@' and source:sub(2) or nil
+end
+
+local function sourcedir(lvl)
+    local source = sourcepath(lvl and lvl + 1 or 3)
+    return source and source:match('(.*)/') or '.'
+end
+
+local mt = getmetatable(debug) or {}
```

Comment:
I just wanted to make no assumptions about it at this point. How do you know that there is no metatable set when registering the module at C level?

Reply:
The code executes during tarantool loading, before any user code. Even if it would be possible to set a metatable this code anyway just replace __index.

Reply:
✔️